lgn-auth/src/jwt.rs

```diff
@@ -44,14 +45,14 @@ impl JWTAuth {
         // <https://github.com/mikkyang/rust-jwt/blob/master/src/lib.rs#L164>
 
         let json_bytes = serde_json::to_vec(&self)?;
-        Ok(base64::encode_config(json_bytes, base64::URL_SAFE_NO_PAD))
```

Comment:
The base64 API changed: https://github.com/marshallpierce/rust-base64/blob/master/RELEASE-NOTES.md#migration